src/backend/api/Fusion.Resources.Domain/Commands/Tasks/GetMultipleRequestTasks.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Fusion.Resources.Database;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Domain.Commands.Tasks
+{
+    public class GetMultipleRequestTasks : IRequest<ILookup<Guid, QueryRequestTask>>
```

Comment:
Maybe use `GetTasksForRequests`? 
The current name to me would indicate i fetch a type of requestTasks and the ids i provide would be the task ids...

Reply:
I also think we should start writing /// comments above the commands/requests to explain what they do, requirements and caveats...